index.bs

```diff
@@ -149,9 +197,19 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/#
         capabilities by a {{MediaEncodingConfiguration}} dictionary.
       </p>
       <p>
-        For a {{MediaConfiguration}} to be a <dfn>valid
-        MediaConfiguration</dfn>, <code>audio</code> or <code>video</code> MUST
-        be <a>present</a>.
+        For a {{MediaConfiguration}} to be a <dfn>valid MediaConfiguration
+        </dfn>, <code>audio</code> or <code>video</code> MUST be <a>present</a>.
+      </p>
+      <p>
+        For a {{MediaDecodingConfiguration}} to describe <a>encrypted media</a>
+        [[!ENCRYPTED-MEDIA]], a {{keySystemConfiguration}} MUST be 
+        <a>present</a>.
+      </p>
+
+      <p class='note'>
+        {{decodingInfo()}} may only be called with a {{keySystemConfiguration}}
+        <a>present</a> within [[secure-contexts]] and is not available from
+        Workers (FIXME: link? issue?).
```

Comment:
Searching for a way to formally determine worker vs non worker context. I can take reference on https://www.w3.org/TR/workers, but not sure if theres an algorithm to actually check this